draft-irtf-panrg-path-properties.md

```diff
@@ -196,9 +200,13 @@ Service function:
 : A service function that a path element applies to a flow, see {{RFC7665}}. Examples of abstract service functions include firewalls, Network Address Translation (NAT), and TCP optimizers. Some stateful service functions, such as NAT, need to observe the same flow in both directions, e.g., by being an element of both the path and the reverse path.
 
 Transparency:
-: A node is transparent with respect to a protocol header, payload, or both for a specific action if the node performs this action independently of the given (meta-)information.
+: When a node performs an action on a flow, the node is transparent to the flow with respect to some (meta-)information if the node performs this action independently of the given (meta-)information.
+(Meta-)information can for example be the existence of a protocol (header) in a packet or the content of a protocol header, payload, or both.
 Actions can for example be blocking packets or reading and modifying (other protocol) headers or payloads.
-An IP router could be transparent for transport protocol headers such as TCP and UDP, in contrast to a NAT that actively modifies TCP and UDP header information.
+An IP router could be transparent to transport protocol headers such as TCP/UDP but not transparent to IP headers as it actively modifies them.
```

Comment:
Just reading the whole thing again: Why does an IP router actively modify IP headers, whereby the text reads as if that's necessarily always the case?
Wouldn't the router just look up the destination address in its forwarding table and then forward the packet, modifying the Ethernet header for sure, but not the IP header?
Sorry, am I just missing something right now? :sweat_smile:

Reply:
I guess the IP router reads these headers for sure, so the example wouldn't have to change much to say this.

Reply:
I was thinking that the IP router changes certain IP header fields (e.g., TTL, IP checksum), however maybe we can make a better example by considering the action of "forwarding on a certain interface".
What do you think?

```suggestion
An IP router could be transparent to transport protocol headers such as TCP/UDP but not transparent to IP headers since its forwarding behavior depends on the IP headers.
```

Reply:
I like your suggestion. And you're right, "modify IP header" doesn't necessarily mean addresses... but still I think your proposed wording is clearer.